packages/ecs-agent/ecs-agent.spec

```
@@ -86,6 +88,21 @@ Requires: %{_cross_os}amazon-ssm-agent
 %description
 %{summary}.
 
+%package base
+Summary: Base configuration files for the ECS agent
+Requires: %{name}
+
+%description base
+%{summary}.
+
+%package nvidia
```

if you change the name of `base`:
```suggestion
%package nvidia-config
```